torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -802,22 +802,17 @@ def crop_bounding_box(
     height: int,
     width: int,
 ) -> Tuple[torch.Tensor, Tuple[int, int]]:
-    # TODO: Investigate if it makes sense from a performance perspective to have an implementation for every
-    #  BoundingBoxFormat instead of converting back and forth
-    bounding_box = convert_format_bounding_box(
-        bounding_box.clone(), old_format=format, new_format=features.BoundingBoxFormat.XYXY, inplace=True
-    )
+
+    bounding_box = bounding_box.clone()
```

Comment:
Vasilis found that it is generally faster to do one regular operation instead of cloning and a single inplace operation. @vfdev-5 Have you benchmarked that as well?

Reply:
Yes, it faster by ~2us to go directly with `bounding_box = bounding_box.sub(sub)` instead of cloning and `sub_`.

Reply:
I'll update in pad PR

Reply:
Funny, full benchmark shows ~10us improvements :)